firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -124,6 +124,10 @@ static current_control_state current_ctrl_state;
 static PID pid_params;
 
 
+//
+static uint8_t torque_sum;
```

Comment:
However, since this is a global static, you should have it initialized to a known value.
static uint8_t torque_sum = 0;